tests/test_export_wkw_as_tiff.py

```diff
@@ -9,118 +9,195 @@
-                                                 shape=tile_bbox["size"])
+                    correct_image = dataset.read(
+                        off=[
+                            tile_bbox["topleft"][0]
+                            + tile_bbox["size"][0] * x_tile_index,
+                            tile_bbox["topleft"][1]
+                            + tile_bbox["size"][1] * y_tile_index,
+                            tile_bbox["topleft"][2] + data_slice_index,
+                        ],
+                        shape=tile_bbox["size"],
+                    )
 
                     correct_image = np.squeeze(correct_image)
 
-                    assert np.array_equal(correct_image, test_image), f"The tiff file {tiff_path} that was written " \
+                    assert np.array_equal(correct_image, test_image), (
+                        f"The tiff file {tiff_path} that was written "
                         f"is not equal to the original wkw_file."
+                    )
 
 
 def test_export_tiff_stack_tiles_per_dimension():
     destination_path = os.path.join("testoutput", "WT1_wkw_tiles_per_dimension")
-    args_list = ["--source_path", os.path.join("testdata", "WT1_wkw"),
-                 "--destination_path", destination_path,
-                 "--layer_name", "color",
-                 "--name", "test_export",
-                 "--bbox", "0,0,0,100,100,5",
-                 "--mag", "1",
-                 "--tiles_per_dimension", "6,6"]
+    args_list = [
+        "--source_path",
+        os.path.join("testdata", "WT1_wkw"),
+        "--destination_path",
+        destination_path,
+        "--layer_name",
+        "color",
+        "--name",
+        "test_export",
+        "--bbox",
+        "0,0,0,100,100,5",
+        "--mag",
+        "1",
+        "--tiles_per_dimension",
+        "6,6",
+    ]
 
     bbox = {"topleft": [0, 0, 0], "size": [100, 100, 5]}
 
     run(args_list)
 
     tile_bbox = {"topleft": bbox["topleft"], "size": [17, 17, 1]}
-    test_wkw_file_path = os.path.join("testdata", "WT1_wkw", "color", Mag(1).to_layer_name())
+    test_wkw_file_path = os.path.join(
+        "testdata", "WT1_wkw", "color", Mag(1).to_layer_name()
+    )
     with wkw.Dataset.open(test_wkw_file_path) as dataset:
         slice_bbox = bbox
         slice_bbox["size"] = [slice_bbox["size"][0], slice_bbox["size"][1], 1]
         for data_slice_index in range(bbox["size"][2]):
 
             for y_tile_index in range(ceil(bbox["size"][1] / tile_bbox["size"][1])):
-                for x_tile_index in range(ceil(tile_bbox["size"][0] / tile_bbox["size"][0])):
-                    tiff_path = os.path.join(destination_path, f"{data_slice_index + 1}", f"{y_tile_index + 1}", f"{x_tile_index + 1}.tiff")
-
-                    assert os.path.isfile(tiff_path), f"Expected a tiff to be written at: {tiff_path}."
+                for x_tile_index in range(
+                    ceil(tile_bbox["size"][0] / tile_bbox["size"][0])
```

Comment:
Do you know where the format changes come from? Did the black version change?

Reply:
I'm not sure the tests had been formatted by black at all.